src/components/MetadataEditor/MetadataEditor.vue

```diff
@@ -134,46 +164,25 @@
          'Sample_Preparation',
          'Submitted_By',
          'MS_Analysis',
-         'metaspace_options',
        ];
        return without(allSections, ...specialSections);
+     },
+     adductOptions() {
+       return this.possibleAdducts[get(this.value, ['MS_Analysis', 'Polarity']) || 'Positive'];
      }
    },
    methods: {
-     getAugmentedSchema(mdType) {
-       const schema = deriveFullSchema(metadataSchemas[mdType]);
-
-       schema.properties.metaspace_options.properties.Metabolite_Database.smEditorHelp = DatabaseDescriptions;
-
-       // Instead of imperatively updating the schema every time something changes, use getters so that anything
-       // that watches the schema automatically subscribes to the correct data source.
-
-       Object.defineProperty(schema.properties.metaspace_options.properties.Adducts.items, 'enum', {
-         get: () => this.possibleAdducts[get(this.value, ['MS_Analysis', 'Polarity'] || 'Positive')],
-       });
-
-       Object.defineProperty(schema.properties.metaspace_options.properties.Metabolite_Database.items, 'enum', {
-         get: () => this.molecularDatabases,
-       });
-
-       return schema;
-     },
-
```

Comment:
🎉  I'm really glad to remove this block of magic behavior. With Adducts moved out of the form, there's no longer a circular dependency between the form schema (which updated whenever Polarity changed) and form values.